build/linux/installer/conf/telegraf-prom-side-car.conf

```diff
@@ -184,12 +192,12 @@
   # kube_config = "/path/to/kubernetes.config"
 
   ## Use bearer token for authorization. ('bearer_token' takes priority)
-  bearer_token = "/var/run/secrets/kubernetes.io/serviceaccount/token"
+  # bearer_token = "/var/run/secrets/kubernetes.io/serviceaccount/token"
```

Comment:
Should we just remove instead of commenting? Or does it get used somewhere?

Reply:
How was it affecting the customer?

Reply:
> How was it affecting the customer?

> customer app validating this token and this token shouldnt be sent as part of the request since all pod based scraping happens without any auth.